trillian/integration/ct_hammer/main.go

```diff
@@ -50,6 +50,7 @@ var (
 	maxGetEntries     = flag.Int("max_get_entries", 500, "Maximum get-entries request size")
 	maxParallelChains = flag.Int("max_parallel_chains", 2, "Maximum number of chains to add in parallel (will always add at least 1 chain)")
 	limit             = flag.Int("rate_limit", 0, "Maximum rate of requests to an individual log; 0 for no rate limit")
+	ignoreErrors      = flag.Bool("ignore_errors", false, "Whether to fail immediately on any error")
```

Comment:
nit: this flag description is kinda describing the opposite flag :)

Reply:
True; fixed.